api_src/server.js

```diff
@@ -12,11 +12,16 @@ const store = configureStore(sagaMiddleware);
 sagaMiddleware.run(rootSaga)
 const app = express();
 
-store.dispatch(userUploadActions.test("dingus amingus"))
+setTimeout(() => {
+    for (let i = 0; i < 10; i++) {
+        store.dispatch(userUploadActions.test("dingus amingus"))
+        store.dispatch(userUploadActions.test("dingus amingus"))
+    }
+}, 5000);
 console.log("testing my logs lmao")
 
 app.get("/", (req, res) => {

```

Comment:
So how is the webpage going to be served if not via base url

Reply:
dunno, haven't done ssr

Reply:
Don't think it's a matter of ssr. We just need to build before we serve the webpage

Reply:
If that is ssr, excuse the retardary